2018/13xxx/CVE-2018-13380.json

```diff
@@ -15,17 +15,11 @@
                     "product": {
```

Comment:
'https://fortiguard.com/advisory/FG-IR-18-383\r' does not match '^(ftp|http)s?://\\S+$'